cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx

```diff
@@ -53,6 +52,7 @@ const CustomSelect: React.FC<ISelectProps> = ({
 }: ISelectProps) => {
   const onChangeHandler = (event: React.ChangeEvent<HTMLSelectElement>) => {
     const v = event.target.value;
+    // setSelection(v);
```

Comment:
nit: Was this intended? If yes can we remove?

Reply:
Removed